independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2444 @@
+                                  "type": "integer",
+                                  "format": "int32",
+                                  "description": "interval shift"
+                                },
+                                "intervals_per_hour": {
+                                  "type": "number",
+                                  "description": "intervals per hour"
+                                }
+                              },
+                              "description": "network"
+                            },
+                            "fueltech": {
+                              "type": "object",
+                              "properties": {
+                                "code": {
+                                  "type": "string",
+                                  "description": "code"
+                                },
+                                "label": {
+                                  "type": "string",
+                                  "description": "label"
+                                },
+                                "renewable": {
+                                  "type": "boolean",
+                                  "description": "renewable"
+                                }
+                              },
+                              "description": "fueltech"
+                            },
+                            "status": {
+                              "type": "object",
+                              "properties": {
+                                "code": {
+                                  "type": "string",
+                                  "description": "code"
+                                },
+                                "label": {
+                                  "type": "string",
+                                  "description": "label"
+                                }
+                              },
+                              "description": "status"
+                            },
+                            "station_id": {
+                              "type": "integer",
+                              "format": "int32",
+                              "description": "Station Id"
+                            },
+                            "code": {
+                              "type": "string",
+                              "description": "code"
+                            },
+                            "dispatch_type": {
+                              "type": "string",
+                              "description": "dispatch type"
+                            },
+                            "active": {
+                              "type": "boolean",
+                              "description": "active"
+                            },
+                            "capacity_registered": {
+                              "type": "number",
+                              "description": "capacity registered"

```

Comment:
@paulculmsee Few description values are Camel cased and a few are not in the swagger, Can you please follow a single pattern for all the description values of all the properties.

Reply:
Part of a problem on my end. Co2 Signal is another connector not meant to be part of this PR